client/my-sites/beta-testing/style.scss

```diff
@@ -0,0 +1,205 @@
+@import '@automattic/onboarding/styles/mixins';
+
+body.is-section-beta-testing {
```

Comment:
I think this rule may be the issue re: @olaolusoga's note on [the white navigation here](https://github.com/Automattic/wp-calypso/pull/54346#issuecomment-883838504).

Comparing with the `/help` page you'll notice that it's set to `rgb(246, 247, 247)` there while this rule is explicitly setting it to `#fff`, which is then blending in with the white navigation on the  **Classic Bright** theme.

**Help Page:**
<img width="1422" alt="CleanShot 2021-07-22 at 11 40 56@2x" src="https://user-images.githubusercontent.com/481776/126667840-5f11e3ee-7dab-4665-9dcb-7d6addde2324.png">

Beta Testing Page:
<img width="1422" alt="CleanShot 2021-07-22 at 11 40 56@2x" src="https://user-images.githubusercontent.com/481776/126667983-3fd2a39a-8d4a-4e52-8996-5b8049cb6cd0.png">

Would it be possible to remove this and, instead, apply the white background to the element with the `#primary` / `.layout__primary`?

**Example**
<img width="1420" alt="CleanShot 2021-07-22 at 11 44 06@2x" src="https://user-images.githubusercontent.com/481776/126668629-90893cbe-a8f8-44bd-bafe-49bfbbecb12c.png">

Reply:
I don't have a strong opinion on this, but we did have a previous design iteration which looked similar.

<img width="480" alt="Screenshot on 2021-07-15 at 11-30-46" src="https://user-images.githubusercontent.com/2749938/126772178-23ac401e-9e78-4e7e-9a80-8fcfff136c07.png">

Happy to return to that version (with adjustments) if this is the desired approach.

Reply:
Oh, I wasn't aware of this version. Personally, I think this iteration is better and it solves the white background issue on the Classic Bright theme. But I'll defer to @olaolusoga on this for the final call!

I'd also be in favor of merging this and readdressing the design as a follow-up, if we're not certain/clear on what option we'd prefer.

Reply:
@claudiucelfilip I spoke w/ @olaolusoga earlier today and he advised that white background is preferred. He also advised adding a border as briefly discussed above.

I recommend adding the border to the right side of the navigation when the navigation is white, based on the active theme (which I believe only occurs on the Classic Bright theme), as opposed to the content area. This way we don't have to worry about addressing multiple pages/components.

We can circle back and look at this in a follow-up. Let's go ahead and ship this as-is, when the translations complete.